framework_tvb/tvb/adapters/creators/local_connectivity_creator.py

```diff
@@ -68,7 +68,7 @@ def get_filters():
 
     @using_template('spatial/spatial_fragment')
```

Comment:
Here the same wrong dependency was maintained. 
tvb.interfaces should depend on adapters, not viceversa.
See tvb-root/framework_tvb/README.rst for the full tree of dependencies